### DISCUSSION TITLE
More of a queston than an issue about migration.

### DISCUSSION THREAD

**Discussion by vvstubbs** [ZoeyVid/NPMplus#1053] [Q&A] [answered]
I have the original NPM running, using a MySQL database instead of  the SQLite, since the first time I use NPM the SQLite database got corrupted. but that's beside the point. I understand that for the migration to NPMplus you point the new install to the letsencrypt directory, that's fine  but my questions are, do you have the old NPM container still running? Dose the new install access the database for all the site configurations?

**Comment by Zoey2936** [comment]
what do you mean exactly?

**Comment by vvstubbs** [comment]
Okay I want to migrate to NPMplus, I have NPM on the same host that NPMplus will be so point the NPMplus docker install to the letsencrypt directory not a problem. My NPM install uses a MySQL database located on a separate host but the NPM compose file is configured to access the remote database. I want to know how the NPMplus deployment know about all 91 configured proxy hosts? Does it use access to the database, or by some other means, (only by the letsencrypt directory?), and if it is by the database the fact that it is a MySQL database going to be an issue.

The other thing I'm not clear on to I keep the original NPM container running during the migration or stop it and the NPMplus container figures it all out. I'm trying to minimize as many "issues" that I may run into. Hope you don't mind

**Reply by Zoey2936** [reply]
sorry, but I still don't get what you question is

**Comment by Zoey2936** [comment, marked as answer]
Since I don't understand your exact question, I can try to explain how things work:
1. the start.sh script copies files (certs/configs/etc.) and edits configs using sed - migration of certs is currently broken, I will release a fix soon until then please use develop for migration
2. The old datebase is unchanged used. MariaDB should still work, but is untested
3. The old npm container is not needed, it can be stopped while/after migrating, only files/folders and database is needed
